src/ApiRequestPanel.js

```diff
@@ -281,6 +281,20 @@ export class ApiRequestPanel extends EventsTargetMixin(LitElement) {
     this._selectedChanged(value);
   }
 
+  get baseUri() {
+    return this._baseUri;
+  }
+
+  set baseUri(value) {
+    const old = this._baseUri;
+    /* istanbul ignore if */
+    if (old === value) {
+      return;
+    }
+    this._baseUri = value;
+    this.requestUpdate('baseUri', old);
```

Comment:
This is what you meant. This is done by LitElement automatically because `baseUri` is defined in the `properties`. This setter is not necessary. You only need to call `requestUpdate` when you override setter for a property.